LoopKitUI/InsulinModelSettings+LoopKitUI.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  InsulinModelSettings+Loop.swift
+//  LoopKitUI
+//
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+
+
+public extension InsulinModelSettings {
+    var title: String {
+        switch self {
+        case .exponentialPreset(let model):
+            return model.title
+        case .walsh(let model):
+            return model.title
+        }
+    }
+    var subtitle: String {
+        switch self {
+        case .exponentialPreset(let model):
+            return model.subtitle
+        case .walsh(let model):
+            return model.subtitle
+        }
+    }
+}
+
+
+public extension ExponentialInsulinModelPreset {
+    var title: String {
+        switch self {
+        case .humalogNovologAdult:
+            return NSLocalizedString("Rapid-Acting – Adults", comment: "Title of insulin model preset")
+        case .humalogNovologChild:
+            return NSLocalizedString("Rapid-Acting – Children", comment: "Title of insulin model preset")
+        case .fiasp:
+            return NSLocalizedString("Fiasp", comment: "Title of insulin model preset")
+        }
+    }
+
+    var subtitle: String {
+        switch self {
+        case .humalogNovologAdult:
+            return NSLocalizedString("A model based on the published absorption of Humalog, Novolog, and Apidra insulin in adults.", comment: "Subtitle of Rapid-Acting – Adult preset")
+        case .humalogNovologChild:
+            return NSLocalizedString("An adjustment to the adult model based on empirical effects in children.", comment: "Subtitle of Rapid-Acting – Children preset")
+        case .fiasp:
+            return NSLocalizedString("A model based on the published absorption of Fiasp insulin.", comment: "Subtitle of Fiasp preset")
+        }
+    }
+}
```

Comment:
I think these strings are now dead with the redo of the insulin model settings screen

Reply:
Actually, it looks like the updated strings from InsulinModelSelection were lost in this transition.

Reply:
Also: having moved from Loop to LoopKit, these should be LocalizedString rather than NSLocalizedStrign now

Reply:
ick I was afraid of that.  Sorry.  I will put the text from the prior InsulinModelSelection in, but also note that I've requested updated text from Melissa Lee in [IFU-24](https://tidepool.atlassian.net/browse/IFU-24?focusedCommentId=25836)